tests/testthat/test-annotate.r

```diff
@@ -48,3 +48,9 @@ test_that("annotation_* has dummy data assigned and don't inherit aes", {
   expect_false(map$inherit.aes)
   expect_false(raster$inherit.aes)
 })
+
+test_that("unsupported geoms signal a warning (#4719)", {
+  expect_warning(annotate("abline", slope = 1), "not support")
```

Comment:
I'd suggest a single `expect_snapshot_warning()` here so that we can see the  complete generated warning.

Reply:
I'll update that.

Reply:
Just to check, did you mean just a test for a single geom type, or a single test with all three warnings? I couldn't figure out how to do the latter.

Reply:
Oh also I realized I had an older version of testthat that didn't have the warning snapshot test. I'll update this again.